bitnami/external-dns/values.yaml

```diff
@@ -1110,3 +1110,11 @@ metrics:
     ## @param  metrics.googlePodMonitor.endpoint The endpoint for Google Managed Prometheus scraping the metrics
     ##
     endpoint: /metrics
+
+
+## Chart Validation
```

Comment:
This is just a nitpick on my part but I would put this param before the providers ones so it gets more visibility.

Reply:
Moved this up, thanks for the suggestion!